src/commcare_cloud/ansible/group_vars/all.yml

```diff
@@ -76,34 +76,28 @@ old_s3_bulk_delete_chunksize: ~
 s3_blob_db_access_key: "{{ s3_access_key|default('') }}"
 s3_blob_db_secret_key:  "{{ s3_secret_key|default('') }}"
 
-datadog_integration_airflow: false
 datadog_integration_couch: false
 datadog_integration_elastic: false
 datadog_integration_gunicorn: false
 datadog_integration_kafka: false
 datadog_integration_kafka_consumer: false
 datadog_integration_nginx: false
 datadog_integration_pgbouncer: false
-datadog_integration_pgbouncer_custom: false
 datadog_integration_postgres: false
 datadog_integration_postgres_receiver: false
 datadog_integration_rabbitmq: false
 datadog_integration_redisdb: false
 datadog_integration_zk: false
-datadog_integration_jmx: false
-datadog_integration_vmware: false
 datadog_integration_http: false
 datadog_integration_haproxy: false
 datadog_integration_tcp_check: false
 datadog_integration_disk_check: true
-datadog_swift_instances: []
 datadog_extra_host_checks: []
 datadog_patch_modules:
   - requests:true
   - gevent:true
   - psycopg:true
   - redis:true
-  - sqlalchemy:true
```

Comment:
Why is this being dropped? HQ still uses `sqlalchemy`

Reply:
It was listed only as a dependency for the `airflow` integration, but I see we set these in the webworkers supervisor configuration.
I'll revert this change.